modules/UI/util/UIUtil.js

```diff
@@ -103,13 +127,13 @@ import KeyboardShortcut from '../../keyboardshortcut/keyboardshortcut';
             'top-right': 'sw'
         };
 
-        element.setAttribute("data-i18n", "[content]" + key);
-        APP.translation.translateElement($(element));
+        $(element).each(function () {
+            var el = $(this);
```

Comment:
It's just a habit to unify selectors, nodes and instances of jQuery using $() wrapper. Current code base contains mix of jQuery and pure JS practices, it is not clear whether I should prefer pure JS. Next time I'll try to avoid jQuery wrappers. Your suggestion is applied.